src/sentry/models/useremail.py

```diff
@@ -32,6 +32,12 @@ def get_primary_email(self, user: "User") -> "UserEmail":
         user_email, _ = self.get_or_create(user=user, email=user.email)
         return user_email
 
+    def get_users_by_id(self, email: str) -> Mapping[int, "User"]:
```

Comment:
Same here, I guess I don't understand adding functions that aren't used yet or what they have to do with webhooks?